src/auth/components/login/Login.module.css

```diff
@@ -4,6 +4,9 @@
   align-items: center;
   flex-direction: row;
   flex: 1;
+  background-color: var(--color-black-ultra-light);
+  /* footer koros is 14px high and "bleeds" bg color. This shifts the koros up and hides the bleed */
+  margin-bottom: -14px;
```

Comment:
Login page has this negative margin to put the background color "between the waves" of the footer koros. But the profile information page doesn't have this. There the "between the waves" is white, while the bottommost parts (delete profile) of the main content background color is light gray.

Reply:
Fixed! Moved the margin to base element of all content areas